tests/v2/test_waku_rln_relay.nim

```diff
@@ -166,6 +163,34 @@ procSuite "Waku rln relay":
     await web3.close()
     debug "disconnected from", EthClient
 
+  asyncTest "registration procedure":
+    # deploy the contract
+    let contractAddress = await uploadContract(EthClient)
+
+    # prepare rln-relay peer inputs
+    let 
+      web3 = await newWeb3(EthClient)
+      accounts = await web3.provider.eth_accounts()
+      # choose one of the existing account for the rln-relay peer  
+      ethAccountAddress = accounts[9]
+    await web3.close()
+
+    # generate the membership keys
+    let membershipKeyPair = membershipKeyGen()
+       #await createEthAccount(EthClientAddress, 100.u256)
+    check:
+      membershipKeyPair.isSome
+
+    # initialize the RLNRelayPeer
+    var rlnPeer = RLNRelayPeer(membershipKeyPair: membershipKeyPair.get(),
+      ethClientAddress: EthClient,
+      ethAccountAddress: ethAccountAddress,
+      membershipContractAddress: contractAddress)
+    
+    # register the rln-relay peer to the membership contract
+    let status = await rlnPeer.register()
+    check:
+      status
```

Comment:
I'd probably make this more explicit, or prase it as a boolean variable like `is_successful`